src/main/kotlin/bandcampcollectiondownloader/BandcampCollectionDownloader.kt

```diff
@@ -114,19 +113,26 @@ object BandcampCollectionDownloader {
         var releasetitle = digitalItem.title
         var artist = digitalItem.artist
         val printableReleaseName = """"${digitalItem.title}" by ${digitalItem.artist}"""
-        Util.log("""Found release $printableReleaseName.""")
+        Util.log("""Found release $printableReleaseName (Bandcamp ID: $saleItemId) """)
 
         // Skip preorders
         val dateFormatter = DateTimeFormatterBuilder().parseCaseInsensitive().appendPattern("dd MMM yyyy HH:mm:ss zzz").toFormatter(Locale.ENGLISH)
-        val releaseUTC = ZonedDateTime.parse(digitalItem.package_release_date, dateFormatter).toInstant()
-        if (releaseUTC > Instant.now()) {
-            Util.log("$printableReleaseName is a preorder; skipping for now.")
-            return
+        Util.log(""" Release date (Unformatted): ${digitalItem.package_release_date}  """)
+		if (digitalItem.package_release_date != null ) {
+            val releaseUTC = ZonedDateTime.parse(digitalItem.package_release_date, dateFormatter).toInstant()
+            Util.log(""" Release date: $releaseUTC """)
+            if (releaseUTC > Instant.now()) {
+                Util.log("$printableReleaseName is a preorder; skipping for now.")
+                return
+            }
```

Comment:
All these changes simply add debug logging, right? In that case I would prefer to undo them for now, as I plan to provide proper verbose logging in the future, and meanwhile I prefer not to clutter the default logging.